api/custom_auth/serializers.py

```diff
@@ -18,17 +20,30 @@ class Meta:
 
 
 class CustomUserCreateSerializer(UserCreateSerializer):
+    key = serializers.SerializerMethodField()
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.fields["key"] = serializers.SerializerMethodField()
 
     class Meta(UserCreateSerializer.Meta):
         fields = UserCreateSerializer.Meta.fields + (
             "is_active",
             "marketing_consent_given",
+            "key",
         )
         read_only_fields = ("is_active",)
         write_only_fields = ("sign_up_type",)
+        extra_kwargs = {
+            "email": {
+                "validators": [
+                    UniqueValidator(
+                        queryset=FFAdminUser.objects.all(),
+                        lookup="iexact",
+                        message=djoser_settings.CONSTANTS.messages.CANNOT_CREATE_USER_ERROR,
```

Comment:
How come this changed from `Messages.CANNOT_CREATE_USER_ERROR` ? Possibly so that we can tweak it in the settings object?

Reply:
Yeah, exactly